Akash/html/assign 6/Assignment1.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> </title>
+        <style>
+            div {
+                background-color: tomato;
+                text-align: right;
+            }
+            div a {
+                background-color: tomato;
+                text-align: right;
+            }
+        </style>
+    </head>
+    <body>
+        <div>
+            <h1>MI-9 intern HTML Assesment</h1>
+        </div>
+        <br />
+        <hr />
+        <div class="a"><h2>personal Details</h2></div>
+        <form>
+            <label for="name">First name :</label>
+            <input type="text" name="frist name" placeholder="frist name" /><br />
+            <label for="last">Last name :</label>
+            <input type="text" name="last name" placeholder="last name" /><br />
+            <lable for="Gender">Gender :</lable>
+            <input type="radio" id="male" name="gender" value="male" />
+            <label for="male">Male</label>
+            <input type="radio" id="male" name="gender" value="male" />
+            <label for="male">female</label>
+            <input type="radio" id="male" name="gender" value="male" />
+            <lable for="other">other</lable><br />
+            <label for="name">Age :</label>
+            <input type="radio" />
+            <label type="text">Less then 18</label>
+            <input type="radio" />
+            <label type="number">18-25</label>
+            <input type="radio" />
+            <label type="number">26-40</label>
+        </form>
+        <div>
+            <h2>Academic Detail</h2>
+        </div>
+        <form>
+            <label type="text">Skill set :</label>
+            <input type="checkbox" />
+            <lable type="text">HTML</lable>
+            <input type="checkbox" />
```

Comment:
give id to input field and for  to label 
both id and vattribute given tu label must be same
make this change in all code